inst/templates/tidy_contributing.md

```diff
@@ -0,0 +1,192 @@
+# Contributing to {{{package}}}
+
+
+## What is this?
+
+A `CONTRIBUTING.md` document (such as this one) has instructions for
+contributing to the project. These are guidelines the maintainers would
+like contributors to adhere to, and exist to make the process flow more
+smoothly.
+
+As a contributor you should try to make accepting your code as easy as
+you can, following the contributing docs greatly increases the chance
+your contribution will be accepted.
+
+Some common tidyverse conventions are
+
+  - Add a bullet to `NEWS.md` for each change referencing the issue
+    number and your GitHub username.
+  - Add `Closes #123` at the end of your commit message to automatically
+    close the issue with the PR is merged.
+  - Document functions with
+    [roxygen2](https://cran.r-project.org/web/packages/roxygen2/) and be
+    sure to run `devtools::document()` before submitting.
+
+## Prerequisites
+
+  - For this guide to make sense, you’ll need to be acquainted with Git
+    and GitHub. If you are not, see [Happy Git and GitHub for the
+    useR](http://happygitwithr.com/) by Jenny Bryan.
+
+  - Before you do a pull request, you should always file an issue and
+    make sure someone from the tidyverse team agrees that it’s a
+    problem, and is happy with your basic proposal for fixing it. If
+    you’ve found a bug, first create a minimal
+    [reprex](https://www.tidyverse.org/help/#reprex). We don’t want you
+    to spend a bunch of time on something that we don’t think is a good
+    idea.
+
+There are lots of ways to contribute *other* than by writing code. See
+[Contribute to the tidyverse](https://www.tidyverse.org/contribute/) for
+more on this.
+
+## Making a pull request - overview
+
+  - When in doubt, discuss in an issue before doing lots of work.
```

Comment:
Redundant with something above.

Reply:
Ideally the body gets shorter and more mechanical and then it stands alone, i.e. doesn't need an Overview.